orcoursetrion/tests/test_core.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+"""
+Test core functionality
+"""
+import unittest
+
+import semantic_version
+
+
+class TestCore(unittest.TestCase):
+    """Test common and core components to the package.
+    """
+
+    def test_version(self):
+        """Validate version matches proper format"""
+        # Will raise ValueError if not a semantic version
+        import orcoursetrion
+        semantic_version.Version(orcoursetrion.VERSION)
```

Comment:
Nit: Shouldn't we handle this and assert that there isn't a ValueError, so to keep true with the whole "failures mean your tested code is behaving out of design, but errors mean your tests need to be adapted" guideline?
This is a really cool thing to check for either way.